### DISCUSSION TITLE
Matching JetStream sizing

### DISCUSSION THREAD

**Discussion by leolll** [wireui/wireui#121] [Q&A] [answered]
First, great project!  I was wondering if anyone has a suggestion for using WireUI with Laravel JetStream.  Ideally my WireUI forms would match the look and feel of my Jetstream forms.  I'm thinking I'll need to run `php artisan vendor:publish --tag='wireui.resources'` and then manually edit each component.

Any other ways to do it?

**Comment by leolll** [comment]
You can see Jetstream input on top and WireUi on bottom.  There are differences in font size for the inputs and the focus ring is styled differently.  From what I can tell, WireUI doesn't currently have a way to make it look like Jetstream.  So I'd need to make Jetstream look like WireUI.

![2021-11-03_16-16-20](https://user-images.githubusercontent.com/2875387/140238084-d8a81276-9f82-4514-afbe-020ad5747776.jpg)

**Comment by ph7jack** [comment, marked as answer]
@leolll Thanks

Yes, the wireui style is different from Jetstream

Two simple ways to solve this:
1. Change all jetstream input to wireui input (the WireUI input is fully compatible)
2. Extends the input component and change the style classes

**Reply by leolll** [reply]
Okay thanks.
